src/msgpack/Converters/MapConverter.cs

```diff
@@ -54,7 +54,7 @@ public override TMap Read(IMsgPackReader reader, MsgPackContext context, Func<TM
         private bool TryGetLengthFromFixMap(DataTypes type, out uint length)
         {
             length = type - DataTypes.FixMap;
-            return (type & DataTypes.FixMap) == DataTypes.FixMap;
```

Comment:
There were a bug:
if type == DataType.Null,
TryGetLengthFromFixMap returns true, and instead of null reading, we are trying to read non-empty fix map.